ui/apps/platform/src/Containers/SystemHealth/UsageStatistics/ShowProductUsage.tsx

```diff
@@ -18,7 +18,7 @@ function ShowProductUsage(): ReactElement {
             </Button>
             <Modal
                 title="Product usage"
-                description="The page shows the collected product usage data. The current usage is computed from the last metrics received from sensors, and can be accurate to about 5 minutes. The maximum usage is aggregated hourly."
+                description="The page shows the collected product usage data: number of secured Kubernetes nodes and CPU units. The current usage is computed from the last metrics received from sensors, and can be accurate to about 5 minutes. The maximum usage is aggregated hourly and only includes clusters which are still connected. The date range is inclusive and depends on the user's timezone. Data shown is not sent anywhere to Red Hat or metrics collection."
```

Comment:
CPU units = central processing unit units

Reply:
last metrics -> most recent metrics?

Reply:
"The page shows the collected product usage data" - does that mean that some secured clusters are there but are not used? I think not :)

How about: "This page shows product size data:"?

Reply:
"The maximum usage" -> The maximum size

Reply:
Last thought: it's been a while since I worked with "Bug in writing" form Lynn Dupre - I warmly recommend this book!. Now ChatGPT is good at making texts readable ;)